### PR TITLE
LineParser.py: Change seperate to separate

### PR DESCRIPTION
**Pull request by stevemontana1980**
Fixes https://github.com/coala-analyzer/coala/issues/2496

### CODE REVIEW

File: coalib/parsing/LineParser.py [hunk 1]
```diff
@@ -89,15 +89,15 @@ def parse(self, line):
...
 
     @staticmethod
-    def __seperate_by_first_occurrence(string,
+    def __separate_by_first_occurrence(string,
                                        delimiters,
                                        strip_delim=False,
                                        return_second_part_nonempty=False):
         """
-        Seperates a string by the first of all given delimiters. Any whitespace
+        separates a string by the first of all given delimiters. Any whitespace
```
Comment (sils1297) [inline coalib/parsing/LineParser.py]: Why do you make the s lower case?

**Comment by sils1297** [comment]
hey, can you accept the community invitation?

File: coalib/parsing/LineParser.py (hunk 1 above)
Comment (stevemontana1980) [inline coalib/parsing/LineParser.py]: My bad, I used change/replace

**Comment by stevemontana1980** [comment]
Done! I hadn't seen it. 

----- Original Message -----

**Comment by sils1297** [comment]
oh can you replace the `Fixes` with a `Closes` in the commit message as this is not a bug? That's relevant for our bug prediction.

**Comment by stevemontana1980** [comment]
Changed Fixes to Closes, do you want me to change it in the PR message as well?

**Comment by Adrianzatreanu** [comment]
ack a2e3ba5

**Comment by Adrianzatreanu** [comment]
@rultor merge

**Pull request merged**